job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -88,7 +88,7 @@ pipeline{
     
         // RHBOP
         ${rhbopNightlyStage(RHBOP_NEXT_PRODUCT_BRANCH, RHBOP_NEXT_PRODUCT_CONFIG_BRANCH)}
-        ${rhbopNightlyStage(RHBOP_CURRENT_PRODUCT_BRANCH, RHBOP_CURRENT_PRODUCT_CONFIG_BRANCH, RHBOP_CURRENT_PRODUCT_VERSION, RHBOP_CURRENT_DROOLS_VERSION)}
+        //{rhbopNightlyStage(RHBOP_CURRENT_PRODUCT_BRANCH, RHBOP_CURRENT_PRODUCT_CONFIG_BRANCH, RHBOP_CURRENT_PRODUCT_VERSION, RHBOP_CURRENT_DROOLS_VERSION)}
```

Comment:
Hey @Ginxo I do not think it works in this way, it will try to execute what is inside `${}`.. I remember it happened before and the only way to fix it was to remove the `$`

Reply:
yeah, we need to add the // and actually remove the $, otherwise it doesnt work.